pages/index.tsx

```diff
@@ -260,7 +260,7 @@ const InstallSection = () => {
     <div key="powershell" className="my-4 text-gray-700">
       <p className="mb-2">PowerShell (Windows):</p>
       <CodeBlock
-        language="bash"
+        language="powershell"
```

Comment:
This doesn't look improving the rendering of these code blocks, rather it seems introducing inconsistency of the `$` sign before each line. I'd prefer to keep these as is.

Reply:
I didn't notice this, makes sense!